agent.go

```diff
@@ -440,6 +443,26 @@ func (s *sandbox) initChannel() error {
 	return s.channel.setup()
 }
 
+func grpcTracer(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp interface{}, err error) {
```

Comment:
Please let the code breathes a little bit into this function, otherwise the code is harder to read.